XmlFormatter/src/Settings/Provider/DataStructure/SerializableScope.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+
+namespace XmlFormatter.src.Settings.Provider.DataStructure
+{
+    /// <summary>
+    /// This class is a data container to for the xml saving
+    /// </summary>
+    public class SerializableScope
+    {
+        /// <summary>
+        /// Public access to the scope name
+        /// </summary>
+        public string Name;
+
+        /// <summary>
+        /// Public access to the namespace of the class this container was created from
+        /// </summary>
+        public string ClassType;
+
+        /// <summary>
+        /// Public access to the sub scopes
+        /// </summary>
+        public List<SerializableScope> SubScopes;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5052706)